com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -331,6 +331,15 @@ public void NetworkShow(ulong clientId)
                 throw new VisibilityChangeException("The object is already visible");
             }
 
+            if (CheckObjectVisibility != null && !CheckObjectVisibility(clientId))
+            {
+                if (NetworkManager.LogLevel == LogLevel.Developer)
```

Comment:
Maybe make this message log if `NetworkManager.LogLevel <= LogLevel.Normal`?
A user would be more likely to see this message and it wouldn't be something that would get spammed all of the time.

Reply:
+1